src/DTS/AbstractEntity.php

```diff
@@ -4,10 +4,10 @@
 namespace HomeCEU\DTS;
 
 
+use HomeCEU\DTS\Entity\GetPropertyTrait;
+
 abstract class AbstractEntity implements Entity {
-  public function get(string $property) {
-    return $this->{$property} ?? null;
-  }
+  use GetPropertyTrait;
```

Comment:
I hate traits

Reply:
I don't mind them so much in tests (I actually just made a couple for some tests that shouldn't share a base class, but does need the same code multiple times.)

I couldn't think of a cleaner way to do this at the time.

Reply:
oh, lol, you didn't have to remove it... I don't like them, but they sometimes are useful... just don't abuse them

Reply:
It was late when I added it. I had added a couple for testing and was just on a roll. I looked back through and found a better way to handle it.